docs/contributing/pull-requests/branching.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Branching
+
+## Naming Convention
+
+To keep branches organized we adhere to a naming convention for branches. The convention splits
+branch names into 2 components, each separated by a `/`.
+
+1. The developers' pod abbreviation (e.g. `sm`)
+2. The Jira issue tag (e.g. `SM-1234`)
+
+```bash
+# Some examples
+sm/SM-1234
+ps/PS-4321-optional-description
+```
+
+This naming convention allows us to easily identify the type of work being done on a branch and will
+assist in identifying and tracking down stale branches.
+
+## Branching for Development
+
+The branching strategy for development depends upon whether the work will be managed using what we
+have termed a "Long-Lived Feature Branch" or not.
+
+:::note
+
+We use the term "feature" in this documentation to refer to any change to the codebase, whether it
+be for a new set of functionality, a bug fix, or refactoring existing code. It is not meant to
+exclude types of development that are not "features" per se.
+
+:::
+
+### Which Branching Model to Choose?
+
+The choice of development branching model depends on how you plan to handle the **entire testable,
+releasable feature**. As such, it should be planned in advance when beginning work on a new set of
+Jira stories or tasks, and it requires coordination across the entire team - Development, QA, and
+Product.
+
+Choose a **Long-Lived Feature Branch** if the feature will:
+
+- Have multiple authors
+- Consist of multiple Pull Requests, which have already been reviewed
+- Require these multiple Pull Requests from multiple authors in order to produce the testable,
+  releasable change
+
+Choose a **Short-Lived Feature Branch** if the feature will:
+
+- Have a single author
+- Consist of a single Pull Request from the Feature Branch into `master` to produce the testable,
+  releasable change
+
+:::tip Still Unsure?
+
+If in doubt, lean toward creating shorter-lived feature branches directly off of `master` for each
+developer’s work, as there is overhead built in to the Long-Lived Feature Branch that can be avoided
+if smaller features can be tested and released independently.
+
+:::
+
+### Long-Lived Feature Branch
+
+A Long-Lived Feature Branch differs from a Short-Lived Feature Branch in three significant ways. It
+should:
+
+- Have multiple authors
+- Consist of multiple Pull Requests, each of which have already been reviewed
+- Only consist of merge commits
```

Comment:
This section seems a bit repetitive of the _Which Branching Model to Choose?_ section.

Reply:
I clarified this, as it was very repetitive.